tests/testthat/test-summariseRecordCount.R

```diff
@@ -200,8 +201,8 @@ test_that("summariseRecordCount() sex argument works", {
 
 
 
-# omopTable <- cdm$condition_occurrence
+# omopTable <- cdm$observation_period
```

Comment:
I would try no not have this in the testing files, we could have an extra folder that is ignored when building the package, but where you can save mock scripts that you use to build it @martaalcalde